### PR TITLE
use an oneliner to install phantomjs on travis

### PR DESCRIPTION
**Pull request by timoschilling**
inspired by @deivid-rodriguez

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -9,7 +9,7 @@ bundler_args: --without development
 cache: bundler
 
 before_install:
-  - source script/install_phantomjs.sh
+  - export PATH=$(npm bin):$PATH && if [ $(phantomjs --version) != '2.1.1' ]; then npm install phantomjs-prebuilt; fi
   - gem update --system # use the very latest Rubygems
```
Comment (varyonic) [inline .travis.yml]: phantomjs-prebuilt@2.1 perhaps?

**Comment by deivid-rodriguez** [comment]
Any news @timoschilling?

**Comment by Fivell** [comment]
@timoschilling , such kind of commits might be better to squash

**Comment by timoschilling** [comment]
@Fivell you are right, next time

**Pull request merged**